frontend/src/metabase/admin/permissions/components/PermissionsEditor/PermissionsEditorLegacyWarning.tsx

```diff
@@ -2,14 +2,18 @@ import { t } from "ttag";
 
 import { useToggle } from "metabase/hooks/use-toggle";
 import { colors } from "metabase/lib/colors";
-import { Icon, Alert, Anchor, Text, Box } from "metabase/ui";
+import { Icon, Alert, Anchor, Text, Box, useMantineTheme } from "metabase/ui";
 
 export const PermissionsEditorLegacyNoSelfServiceWarning = () => {
   const [isExpanded, { toggle }] = useToggle(false);
+  const theme = useMantineTheme();
 
   return (
     <Box mt="md" mb="sm" style={{ marginInlineEnd: "2.5rem" }}>
-      <Alert icon={<Icon name="warning" size={16} />} color="accent5">
+      <Alert
+        icon={<Icon name="warning" size={16} />}
+        color={theme.fn.themeColor("accent5")}
```

Comment:
This change is to allow typing `CUSTOM_COLORS` in `frontend/src/metabase/ui/utils/colors.ts` to be easier since `accent4` and `accent5` is only used once in Mantine context.